src/analysis/retail/druid/balance/modules/spells/WaningTwilight.tsx

```diff
@@ -0,0 +1,118 @@
+import { formatNumber, formatPercentage } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import { calculateEffectiveDamage } from 'parser/core/EventCalculateLib';
+import Events, { DamageEvent } from 'parser/core/Events';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import { TALENTS_DRUID } from 'common/TALENTS';
+import Enemies from 'parser/shared/modules/Enemies';
+import { RoundedPanel } from 'interface/guide/components/GuideDivs';
+import { explanationAndDataSubsection } from 'interface/guide/components/ExplanationRow';
+import { mergeTimePeriods, OpenTimePeriod } from 'parser/core/mergeTimePeriods';
+import uptimeBarSubStatistic, { SubPercentageStyle } from 'parser/ui/UptimeBarSubStatistic';
+import { TrackedBuffEvent } from 'parser/core/Entity';
+
+const WANING_TWILIGHT_BONUS_DAMAGE = 0.04;
+export const GUIDE_CORE_EXPLANATION_PERCENT = 40;
+const WT_COLOR = '#00bb44';
+
+class WaningTwilight extends Analyzer {
+  static dependencies = {
+    enemies: Enemies,
+  };
+  protected enemies!: Enemies;
+
+  damageGotten = 0;
+  talentRank: number;
```

Comment:
probably would be better to have a variable to store the total percent damage increase and calculate it once in the constructor instead of re-doing the multiplication for each damage event